home-assistant-plugin/custom_components/xknx.py

```diff
@@ -17,7 +17,7 @@
 from homeassistant.helpers.event import async_track_state_change
 from homeassistant.helpers.script import Script
 
-REQUIREMENTS = ['xknx==0.8.5']
+REQUIREMENTS = ['xknx==0.9.0']
```

Comment:
please remove this from the PR. The current version will be 0.9.3 and we will merge it from HA directly into XKNX.